runtime/util/vmargs.c

```diff
@@ -405,20 +406,24 @@ optionValueOperations(J9PortLibrary *portLibrary, J9VMInitArgs* j9vm_args, IDATA
 				break;
 
 			case GET_OPTION_OPT :
-				/* Note - currently won't work with MAP_WITH_INCLUSIVE_OPTIONS */
-				if ((errorFound = optionValueOperations(PORTLIB, j9vm_args, element, GET_OPTION, &values, 0, delim, 0, NULL)) != OPTION_OK) {		/* get correct values string */
+				if (bufSize > 0)
+					values = (char *)malloc(bufSize);
```

Comment:
This is the not the right approach to modifying the parsing code.  And there's both formatting and incorrect calls happening here